snapcraft/parts/plugins/matter_sdk_plugin.py

```diff
@@ -75,9 +76,41 @@ def __init__(
     ) -> None:
         super().__init__(properties=properties, part_info=part_info)
 
-        self.matter_dir = part_info.part_build_dir
+        self.matter_sdk_dir = part_info.part_build_dir
         self.snap_arch = os.getenv("SNAP_ARCH")
 
+    @overrides
+    def get_pull_commands(self) -> List[str]:
+        options = cast(MatterSdkPluginProperties, self._options)
+        commands = []
+
+        if self.snap_arch == "arm64":
+            commands.extend(
+                [
+                    f"wget --no-verbose {ZAP_REPO}/releases/download/"
+                    f"{options.matter_sdk_zap_version}/zap-linux-{self.snap_arch}.zip",
+                    f"unzip -o zap-linux-{self.snap_arch}.zip -d zap",
+                    "echo 'export ZAP_INSTALL_PATH=$PWD/zap'",
```

Comment:
Exporting ZAP_INSTALL_PATH here is useless.

Reply:
It has now been updated to be exported directly to the matter_sdk_env' file